src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -41,6 +41,8 @@
  * {@link #createWorkUnit}
  * {@link #createGSONWorkUnit}
  *
+ * This requires that the static {@link #start(RootDoc)} methods is implemented to call the
+ * {@link #startProcessDocs(RootDoc)} from their instance.
```

Comment:
I would remove the blank line (43) so this appears with the list right above, and change it to say:

{@link #start(RootDoc)} A static method that instantiates the subclass and delegates to the instance method {@link #startProcessDocs(RootDoc)}.